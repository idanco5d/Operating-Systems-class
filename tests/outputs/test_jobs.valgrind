==1823== Memcheck, a memory error detector
==1823== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==1823== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==1823== Command: /mnt/c/Users/illus/os/hw/1/smash
==1823== Parent PID: 1818
==1823== 
==1823== 
==1823== FILE DESCRIPTORS: 4 open (3 std) at exit.
==1823== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_jobs.valgrind
==1823==    <inherited from parent>
==1823== 
==1823== 
==1823== HEAP SUMMARY:
==1823==     in use at exit: 0 bytes in 0 blocks
==1823==   total heap usage: 40 allocs, 40 frees, 80,266 bytes allocated
==1823== 
==1823== All heap blocks were freed -- no leaks are possible
==1823== 
==1823== For lists of detected and suppressed errors, rerun with: -s
==1823== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
