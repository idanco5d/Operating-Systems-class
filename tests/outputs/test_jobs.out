smash> smash> smash> [1] sleep 100& : 9315 0 secs
[2] sleep 200& : 9317 0 secs
smash> 